pom.xml

```diff
@@ -26,7 +26,7 @@
 
   <groupId>io.fabric8.elasticsearch</groupId>
   <artifactId>openshift-elasticsearch-plugin</artifactId>
-  <version>2.4.4.13-SNAPSHOT</version>
+  <version>2.4.4.14-SNAPSHOT</version>
```

Comment:
I guess you added this for testing with origin-aggregated-logging?  This will have to be removed before being committed, or it will break the jenkins release job.

Reply:
Well I merged my branch with the master and 2.4.4.14-SNAPSHOT is the latest version https://github.com/fabric8io/openshift-elasticsearch-plugin/blob/master/pom.xml#L29 maybe I missed something?

Reply:
Ah, I see - you didn't make that change - it was changed when you rebased on top of upstream?

Reply:
Yes that's right

Reply:
Please revert this change.  One step of the 'release' workflow will update these values accordingly.